extensions/bitwarden/src/authenticator.tsx

```diff
@@ -301,28 +302,40 @@ const authenticator = {
       if (error) return;
 
       const setTimeAndCode = () => {
-        const timeRemaining = Math.ceil(generator.remaining() / 1000);
-        setTime(timeRemaining);
-
-        if (timeRemaining === generator.period) {
-          setCode(generator.generate());
+        try {
+          const timeRemaining = Math.ceil(generator.remaining() / 1000);
+          setTime(timeRemaining);
+
+          if (timeRemaining === generator.period) {
+            setCode(generator.generate());
+          }
+        } catch {
+          const error = new Error("ERR2: Failed to regenerate");
+          setState(Err(error));
+          captureException(error.message, error, { captureToRaycast: true });
         }
       };
 
-      let interval: NodeJS.Timeout | undefined;
-      // set an initial timeout to ensure the first evaluation is time accurate
-      // and then keep evaluating every second
-      const timeout = setTimeout(() => {
-        setTimeAndCode();
-        interval = setInterval(setTimeAndCode, 1000);
-      }, generator.remaining() % 1000);
-
-      setCode(generator.generate()); // first generation before the interval starts
-
-      return () => {
-        clearTimeout(timeout);
-        clearInterval(interval);
-      };
+      try {
+        let interval: NodeJS.Timeout | undefined;
+        // set an initial timeout to ensure the first evaluation is time accurate
+        // and then keep evaluating every second
+        const timeout = setTimeout(() => {
+          setTimeAndCode();
+          interval = setInterval(setTimeAndCode, 1000);
+        }, generator.remaining() % 1000);
+
+        setCode(generator.generate()); // first generation before the interval starts
+
+        return () => {
+          clearTimeout(timeout);
+          clearInterval(interval);
+        };
+      } catch {
+        const error = new Error("ERR1: Failed to generate");
+        setState(Err(error));
+        captureException(error.message, error, { captureToRaycast: true });
```

Comment:
This will send a simple `ERR1: Failed to generate` error to the Extension Issues dashboard.